doc/INTEGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Axelar Amplifier Integration Guide
```

Comment:
I'm not sure if this is the proper placement for the file. @eguajardo where should this be placed so it renders correctly in the mdbook?

Reply:
file must be inside `doc/src` folder and it should be referenced in the `SUMMARY.md` so it can be properly indexed. You can test it by running:
```
mdbook build doc
mdbook serve doc --open
```